api/v1/user_manager_service.proto

```diff
@@ -166,6 +244,40 @@ service UsersService {
     };
   }
 
+  rpc CreateProject(CreateProjectRequest) returns (Project) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization_id}/projects"
```

Comment:
Given that URL paths are mostly case-insensitive, it might make sense to always have lower-case for org IDs and project IDs.